lib/utils.c

```diff
@@ -39,6 +39,54 @@
 
 int timestamp_short;
 
+int read_prop(char *dev, char *prop, long *value)
+{
+	char fname[41], buf[80], *endp;
```

Comment:
Is there a reason you replaced `IFNAMSIZ+25` with `41`? I think we should keep it the way it was.

Reply:
Actually, I think even `IFNAMSIZ+25` is kind of broken -- really long property names (like `gro_flush_timeout`) will cause the filename to be something like `/sys/class/net/enp3s0f2/gro_flush_timeout` to be over this limit. Let's just make it `char fname[128]`.